docs/key-features/features.md

```diff
@@ -2,4 +2,89 @@
 title: Key Features
 ---
 
-TBD
+## Cross-cloud multi-cluster multi-mode management
```

Comment:
How about add a feature list before introducing the feature?

Reply:
I move the feature list first